### DISCUSSION TITLE
pass-through

### DISCUSSION THREAD

**Discussion by killjoy2013** [apollographql/router#2692] [Q&A] [answered]
is it possible to add pass-through routes in router.yaml? 
for instance, I have a `/login` route which is anonymous & Router should not authenticate those requests.

Thanks

**Comment by Geal** [comment]
hi, do you mean pass-through as in proxying to another node, or as in ignored by the authentication plugin?

**Comment by killjoy2013** [comment]
ignoring the authentication for a specific route actually...

**Reply by Geal** [reply, marked as answer]
there's a change we are planning to do (right after https://github.com/apollographql/router/pull/2672 is merged) where the JWT plugin would only reject requests with invalid (expired, bad signature) tokens, but otherwise let everything go through. The next step will then handle the authorization, like checking scopes, refusing all unauthenticated requests, or as you want, ignore authentication for one path. That will be doable in rhai
